cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,70 @@
+/// <reference types="cypress" />
+
+import { faker } from '@faker-js/faker';
+import { generateUser } from '../support/generateUser'; 
+
 describe('Web Tables page', () => {
-  it('', () => {
+  const newName = faker.name.firstName();
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('Check for pagination', () => {
+    cy.get('.-pagination').should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page').should('exist');
+    cy.get('.-pageSizeOptions').should('exist');
+  });
+
+  it('Check for rows count selection', () => {
+    cy.get('select').select('10 rows');
+  });
+
+  it('Check for adding new worker and search ', () => {
+    cy.createUser(user);
```

Comment:
I have an error user is not defined. You give user argument to function createUser, but this argument is not defined in the code above( try to fix it pls